source/docs/prismic/setup.md

```diff
@@ -0,0 +1,158 @@
+
+```json
+{
+  "Main" : {
+    "uid" : {
+      "type" : "UID",
+      "config" : {
+        "label" : "UID",
+        "placeholder" : "URL used by FC to access the CMS content"
+      }
+    },
+    "meta_title" : {
+      "type" : "Text",
+      "config" : {
+        "label" : "Meta title",
+        "placeholder" : "Meta title for SEO purposes"
+      }
+    },
+    "meta_description" : {
+      "type" : "Text",
+      "config" : {
+        "label" : "Meta description",
+        "placeholder" : "Meta description for SEO purposes"
+      }
+    },
+    "title" : {
+      "type" : "Text",
+      "config" : {
+        "label" : "Title",
+        "placeholder" : "title"
+      }
+    },
+    "content_heading" : {
+      "type" : "StructuredText",
+      "config" : {
+        "single" : "heading1",
+        "label" : "Content heading",
+        "placeholder" : "Title of the page"
+      }
+    },
+    "content" : {
+      "type" : "StructuredText",
+      "config" : {
+        "multi" : "paragraph, preformatted, heading1, heading2, heading3, heading4, heading5, heading6, strong, em, hyperlink, image, embed, list-item, o-list-item, o-list-item",
+        "label" : "Content",
+        "placeholder" : "Content of the page"
+      }
+    }
+  }
+}
+```
+
+And finally, save the type.
+
+### CMS Block
+
+On `http://your-repository.prismic.io/masks/`, click on _Create new_ and create a _Repeatable Type_ named `CMS Block`which _API ID_ is `cms-block`, then in the _JSON editor_ tab in the right pane, paste the following code:
+
+```json
+{
+  "Main": {
+    "uid": {
+      "type": "UID",
+      "config": {
+        "label": "UID",
+        "placeholder": "URL used by FC to access the CMS content"
+      }
+    },
+    "title": {
+      "type": "Text",
+      "config": {
+        "label": "Title",
+        "placeholder": "Title of the page"
+      }
+    },
+    "content": {
+      "type": "StructuredText",
+      "config": {
+        "multi": "paragraph, preformatted, heading1, heading2, heading3, heading4, heading5, heading6, strong, em, hyperlink, image, embed, list-item, o-list-item, o-list-item",
+        "label": "Content",
+        "placeholder": "Content of the page"
+      }
+    }
+  }
+}
+```
+
+And finally, save the type.
+
+## Test it
+
+You can now (re)start Front-Commerce to take your configuration changes into account.
+
+To test it, on Prismic side, you can now create a new `CMS Page` content. When creating the page, take note of the `UID` you set, it will be used as the page identifier so the page will be accessible in Front-Commerce at the URL `http://your-fc-url/the-uid-you-defined`.
+
+<blockquote class="tip">
+If you include images in your content, don't forget to add the domain `images.prismic.io` to `imgSrc` in [the `contentSecurityPolicy` configuration](/docs/reference/configurations.html#config-website-js).
```

Comment:
We do have a media proxy for images: https://gitlab.com/front-commerce/front-commerce-prismic/-/blob/a18608d78a39f2afffc92bb99a8a096116b197b7/prismic/server/express/index.js#L16

Is there a reason why we still need to do so? Don't we proxy images automatically?

Reply:
good question, I completely forgot to check what this is doing

Reply:
so to sum up, we don't proxy images automatically (yet) even if the proxy is in
place, so for now this is needed